src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java

```diff
@@ -451,21 +456,54 @@ protected void containerIsStarting(InspectContainerResponse containerInfo) {
     }
 
     private boolean setupTopology() {
-        try {
-            executeScript(topologyConfigurationFile).get();
-            // The client connection will be closed after that command
-        } catch (Exception e) {
-            if (e instanceof ExecutionException) {
-                if (e.getCause() instanceof TimeoutException) {
-                    return true;
-                    // Do nothing, the cluster is reloading
-                } else if (e.getCause() instanceof TarantoolConnectionException) {
-                    // Probably cluster is not ready
-                    logger().error("Failed to setup topology: {}", e.getMessage());
-                    return false;
+        if (topologyConfigurationFile == null) {
+            String runDirPath = null;
+            try {
+                Container.ExecResult envVariablesContainer = this.execInContainer("env");
```

Comment:
Why you are getting container env ? 
see withArguments()

You can just use  System.getenv() or default to ```./tmp/run```

Reply:
I try use System.getenv() - this doesn't work (haven't need env).
We don't pass run-dir in Argumets. This is env in Dockerfile.
Default to ./tmp/run - would not like use hardcore